apps/agreement/contracts/example/RegistryApp.sol

```diff
@@ -121,23 +122,29 @@ contract Registry is DisputableAragonApp {
     * @notice Schedule a new entry
     * @dev IForwarder interface conformance
     * @param _data Data requested to be registered
+    * @param _context Link to a human-readable text giving context for the given action
     */
-    function forward(bytes memory _data) public {
```

Comment:
Why are we removing the `memory` keyword?

Reply:
It is calldata now that it is external :)

Reply:
Oh, of course, haha! :facepalm: